pyflakes/checker.py

```diff
@@ -2381,9 +2388,13 @@ def EXCEPTHANDLER(self, node):
     def ANNASSIGN(self, node):
         self.handleNode(node.target, node)
         self.handleAnnotation(node.annotation, node)
+        # If the assignment has value, handle the *value* now.
         if node.value:
-            # If the assignment has value, handle the *value* now.
-            self.handleNode(node.value, node)
+            # If the annotation is `TypeAlias`, handle the *value* as an annotation.
+            if _is_name_or_const(node.annotation, 'TypeAlias'):
```

Comment:
this should use `is_typing_helper` which is specifically designed for this

Reply:
thanks, updated